### PR TITLE
Added Discord Webhook Config that was missing

### PR DESCRIPTION
**Pull request by FBFezz**
I think its a WIP. The code of it is in twitter.lua server side at the very bottom

image

### CODE REVIEW

**Comment by DmACKGL** [comment]
https://github.com/manueljlz/gcphone/commit/a6d3102ee187789b06a38f5cb50631863f28c977 Ready for this change.

**Comment by FBFezz** [comment]
Is it supposed to work for twitter?

**Comment by DmACKGL** [comment]
Looks like it does, I'm not going to approve it before I test it.
Let me some minutes to check it.

**Comment by FBFezz** [comment]
Ok, i would check it but currently fixing my guitar

**Comment by DmACKGL** [comment]
Ok, after checking the hook it works. But only when you change the default tweet.authorIcon to a photo hosted outside the server. I'm going to do some research to solve this issue

**Comment by FBFezz** [comment]
Thats weird, also say if i was putting the logo for my twitter profile would it be like this (if i was using imgur) https://imgur.com/gallery/xJBWVwN

**Comment by FBFezz** [comment]
Weird, For some reason i cant get an imgur pic to show up as my profile pic on twitter

**Comment by FBFezz** [comment]
Added your fix to my twitter and fixed the misspelt config.discordwebhook and also fixed the config file

**Comment by DmACKGL** [comment]
New update on the way 
image

**Comment by FBFezz** [comment]
Oh nice, would that require screenshot basic tho?

**Comment by FBFezz** [comment]
If you need me to test anything i have a onesync server that uses Tokovoip

File: resources/gcphone/config.lua [hunk 1]
```diff
@@ -24,4 +24,6 @@ Config.KeyTakeCall  = 38  -- E
 Config.UseMumbleVoIP = false -- Use Frazzle's Mumble-VoIP Resource (Recomended!) https://github.com/FrazzIe/mumble-voip
 Config.UseTokoVoIP   = false
 
-Config.ShowNumberNotification = false -- Show Number or Contact Name when you receive new SMS
\ No newline at end of file
+Config.ShowNumberNotification = false -- Show Number or Contact Name when you receive new SMS
+
+discord_webhook = '' -- Set Discord Webhook (WIP)
```
[the remark this answers is not in the document]
Comment (FBFezz) [reply resources/gcphone/config.lua]: Look at the latest commits i made

**Comment by DmACKGL** [comment]
> Oh nice, would that require screenshot basic tho?

Yep, but we have a public image server with CDN now! (no need to create your own image sever)

**Comment by FBFezz** [comment]
That is sick, Good work and glad you are still doing updates for this. I see this going places :)

**Pull request merged**